plugins/worker/lib/BlockedBy.js

```diff
@@ -208,7 +208,23 @@ async function checkBlockingJob({ jobId, buildId }) {
 
     let blockedBy = this.args[0].blockedBy.split(',').map(jid => `${runningJobsPrefix}${jid}`);
 
-    if (!enforceBlockedBySelf) {
+    const allowSameJob = this.args[0].allowSameJob;
+    const blockTime = this.args[0].blockTime;
+    const json = await this.queueObject.connection.redis.hget(`${queuePrefix}timeoutConfigs`, lastRunningBuildId);
+    const timeoutConfig = JSON.parse(json);
+    let notBlockedBySelf = false;
```

Comment:
feel like `blockedBySelf = true` is easier to read/understand
but maybe just me